app/assets/javascripts/app.js

```diff
@@ -2,16 +2,3 @@
 if (window.history && window.history.replaceState && typeof window.history.replaceState === 'function') {
   window.history.replaceState(null, null, window.location.href);
 }
-
```

Comment:
why has this been deleted?

Reply:
Back button does not work if you do a post with empty/invalid data. This was raised as a bug. Refer the video in the ticket. https://jira.tools.tax.service.gov.uk/browse/TGP-429

I had a chat with @liandra-giacomine about this and we decided to remove this and set the backlink in the layout which can pass from any view